doc/cpu/branch/branch-prediction-fr.md

```diff
@@ -130,20 +132,20 @@ On part d'un tableau d'entier qui contient des nombres aléatoire compris entre
 
 En Java, on pourrait écrire le programme de cette manière :
 ```java
-    int[] array = IntStream.generate(() -> rnd.nextInt() % 1_000)
-        .limit(30_000)
-        .toArray();
-    int[] sortedArray = IntStream.of(input).sorted().toArray();
-
-    long sumPositive(int[] array) {
-        long sum = 0;
-        for (int value : array) {
-          if (value >= 0) {
-            sum += value;
-          }
-        }
-        return sum;
+int[] array = IntStream.generate(() -> rnd.nextInt() % 1_000)
+    .limit(30_000)
+    .toArray();
+int[] sortedArray = IntStream.of(input).sorted().toArray();
```

Comment:
Shouldn't it be `array` rather than `input`?

Reply:
thanks, done :)